examples/mayor/example_response_full.json

```diff
@@ -0,0 +1,184 @@
+{
+  "status": 200,
+  "data": {
+    "last_updated": 1660758114786,
+    "current": {
+      "year": 224,
+      "name": "Derpy",
+      "type": "derp",
+      "perks": [
+        {
+          "name": "TURBO MINIONS!!!",
+          "description": "Minions have double the output!"
+        },
+        {
+          "name": "AH CLOSED!!!",
+          "description": "The Auction House will be closed while Derpy is elected!"
+        },
+        {
+          "name": "DOUBLE MOBS HP!!!",
+          "description": "ALL monsters have double health!"
+        },
+        {
+          "name": "MOAR SKILLZ!!!",
+          "description": "Gain §d+50% §7more skill experience!"
+        }
+      ],
+      "chart": "https://quickchart.io/chart?bkg=rgb(46,49,55)&c={\"type\":\"bar\",\"data\":{\"labels\":[\"Derpy\",\"Cole\",\"Paul\",\"Diaz\",\"Foxy\"],\"datasets\":[{\"label\":\"Votes\",\"data\":[804577,23115,15851,373,94],\"backgroundColor\":[\"rgba(255, 85, 85, 0.8)\",\"rgba(85, 255, 85, 0.8)\",\"rgba(85, 255, 255, 0.8)\",\"rgba(255, 255, 85, 0.8)\",\"rgba(255, 85, 255, 0.8)\"]}]}}",
```

Comment:
So correct me if I'm wrong, but the only difference here from Hypixel's API is the charts?

Reply:
Just realised, yeah ;-;. Should I revert this or?

Reply:
Yeah, might as well. Not much need for it